voxblox/include/voxblox/core/block.h

```diff
@@ -11,7 +12,12 @@
 
 namespace voxblox {
 
-/** An nxnxn container holding VoxelType. It is aware of its 3D position and
+namespace Update {
+/// State of which derived things still need to be updated.
+enum State { kMap, kMesh, kEsdf, kCount };
```

Comment:
Might be a bit nit-picky: The naming `Update` here makes it look like a class.You could define it as a struct. Or directly use an enum class called `UpdateState`. Also I'm not sure "state" makes sense here, at least when I first read this part I was confused what this actually is, and imagined some state-machine like thing where this enum describes what updates you have done already in sequential order. What do you think about `UpdateFlag` or `Update::Flag`?

Reply:
How about Status? Update::Status kind of makes sense to me. The namespace is just to push it so that the constants don't have global scope; i.e., have to refer to it as Update::kMesh rather than voxblox::kMesh. Can also do the same with a class enum wrapper. What do you prefer?

Reply:
sounds good, I would go for an enum class, at least naming wise that would be more consistent with the style guide.